src/main/java/com/example/alaa/university/domain/University.java

```diff
@@ -0,0 +1,94 @@
+package com.example.alaa.university.domain;
+
+import java.time.Instant;
+import java.util.List;
+
+public class University {
+    private Integer id;
+    private String name;
+    private Address address;
+    private UniversityType universityType;
+    private  String email;
+    private  Double studyCost;
+    private Instant startStudyDay;
```

Comment:
According to the name it looks like it should be in Student Class. Maybe you wanted that date be "foundAt" ? so it means when this University was found, and started operating

Reply:
yes ,I  mean started operating